app/src/main/java/com/example/dragonx/NetworkService/Rockets.kt

```diff
@@ -0,0 +1,19 @@
+package com.example.dragonx.NetworkService
+
+import com.google.gson.annotations.SerializedName
+data class Rockets (val rockets: List<Rocket>)
+data class Rocket (
+    var name: String? = null,
+    var description: String? = null,
+    var wikipedia: String? = null,
+    var diameter: Diameter,
+    @SerializedName("dry_mass_kg") var massKg: Int? = null,
+    @SerializedName("dry_mass_lb") var massLb: Int? = null,
+    @SerializedName("first_flight") var firstFlight: String? = null,
+    @SerializedName("flickr_images") var flickrImages: ArrayList<String>? = null,
```

Comment:
- Some of properties named and some are not named, follow single style please. 
- Why do you use var?
- Create dedicated files for classes.
- Try do not use nullable variables.